game/src/test/scala/io/aigar/controller/GameControllerSpec.scala

```diff
@@ -1,38 +1,44 @@
 import io.aigar.controller._
+import io.aigar.controller.response._
+
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
 
 import org.scalatra.test.specs2._
+import org.specs2.matcher._
 
-class GameControlleSpec extends MutableScalatraSpec {
+class GameControlleSpec extends MutableScalatraSpec with JsonMatchers {
+  protected implicit val jsonFormats: Formats = DefaultFormats
   addServlet(classOf[GameController], "/*")
 
-  "GET / on GameController" should {
-    "return status 200" in {
-      get("/") {
-        status must_== 200
-      }
-    }
-  }
-
   "GET /:id on GameController" should {
-    "return status 200" in {
+    "return the right data format" in {
       get("/1") {
         status must_== 200
+
+        parse(body).extract[GameStateResponse] must not(throwAn[MappingException])
       }
     }
   }
 
   "POST / on GameController" should {
-    "return status 200" in {
+    "return the right data format" in {
```

Comment:
Maybe a bit more explicit in the description? E.g. `"return a URL to watch the game"`. I think it helps document what we expect from it. I'm fine if you prefer it this way though.